docs/README.md

```diff
@@ -0,0 +1,14 @@
+# Kubeapps documentation
+
+Everything you need to know about Kubeapps.
+
+## Documentation overview
+
+A high-level overview of how it’s organized will help Kubeapps contributors and users know where to look for certain things.
+
+| Section                      | Description                                                                              |
+| ---------------------------- | ---------------------------------------------------------------------------------------- |
+| [Tutorials:](./tutorials/)   | Start here if you’re new to Kubeapps. A hands-on introduction to Kubeapps for new users. |
+| [How-to guides:](./howto/)   | They guide you through the steps involved in addressing key problems and use-cases.      |
+| [Background:](./background/) | Dive into the overall architecture and implementation details of Kubeapps.               |
+| [Reference:](./reference/)   | Technical information - developer guides, design proposals, examples, translations, etc. |
```

Comment:
Perhaps remove the colon `:` in the first column, as you've done for the other tables in the sub docs?

Reply:
+1. Thanks for the heads up.